Library/Homebrew/style.rb

```diff
@@ -242,6 +261,12 @@ def self.run_shfmt(files, fix: false)
       $CHILD_STATUS.success?
     end
 
+    def self.run_actionlint(files)
+      files = github_workflow_files if files.blank?
+      system actionlint, "-shellcheck", shellcheck, *files
```

Comment:
We probably want to pass `-config-file` with the appropriate `.github/actionlint.yaml` here.

Reply:
@carlocab Can you make a code suggestion? I don't know which is the appropriate `.github/actionlint.yaml` you mean.

Reply:
Will need to think about a code suggestion, since it's a little tricky given that there could be a different `.github/actionlint.yaml` per repository. For example:

https://github.com/Homebrew/brew/blob/91affd113e0c8e853bf5c762fcc479f6638b7212/.github/actionlint.yaml
https://github.com/Homebrew/homebrew-core/blob/7d9e7072ac76169a8a40d910fbbc87231c7eabd2/.github/actionlint.yaml
https://github.com/Homebrew/actions/blob/a6188048cd0171bb4cac956feeff00e588500e7d/.github/actionlint.yaml

We could maybe try to use the same one across all repositories:

```suggestion
      system actionlint, "-shellcheck", shellcheck, "-config-file", HOMEBREW_REPOSITORY/".github/actionlint.yaml", *files
```

Reply:
> We could maybe try to use the same one across all repositories:

Good idea. This seems more fitting with what `brew style` generally does which is to enforce our house style everywhere. Applying!

Reply:
@shivammathur Maybe! It's possible but perhaps not desirable. We don't do this for `.rubocop.yml` or `shellcheck` so would need to hear more before 👍🏻ing it. Can you elaborate?

Reply:
So, now when I run 
```zsh
brew test-bot --only-tap-syntax
```
brew style fails if a tap has self-hosted runners mentioned directly in the workflow as actionlint requires them to be configured in actionlint.yaml.
https://github.com/shivammathur/homebrew-extensions/actions/runs/9507192727/job/26206107036#step:4:41

If it can read the actionlint.yaml from the tap, I can configure those on my end.

Reply:
@MikeMcQuaid 
https://github.com/Homebrew/brew/pull/17506